src/components/common/molecules/SearchBar/styled.tsx

```diff
@@ -1,43 +1,43 @@
-export {};
+import { FaSearch } from 'react-icons/fa';
+import styled from 'styled-components';
 
+const StyledMagnifierIcon = styled(FaSearch).attrs({})`
+  cursor: pointer;
+  &:active {
+    transform: scale(0.9);
+  }
+  color: ${({ theme }) => theme.colors.grey.rockport};
+`;
 
-`
-.box {
+const Wrapper = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: center;
   align-items: center;
-  width: 13rem;
+  background: ${({ theme }) => theme.colors.grey.mist};
+  width: 15rem;
   height: 2rem;
-  border: 2px solid black;
-  border-top-left-radius: 10px;
-  border-bottom-left-radius: 10px;
-  border-top-right-radius: 10px;
-  border-bottom-right-radius: 10px;
-}
+  border: 2px solid ${({ theme }) => theme.colors.grey.deepmist};
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+  border-top-right-radius: 20px;
+  border-bottom-right-radius: 20px;
+`;
 
-.box input {
+const Input = styled.input<{ foo?: string }>`
```

Comment:
Please, remove this type annotation too (the whole `<{ foo?: string }>` thing). It was only there as an example lol

Reply:
I was just keeping this here for reference. Can delete if needed.